lib/std/mem/Allocator.zig

```diff
@@ -10,8 +10,9 @@ const builtin = @import("builtin");
 pub const Error = error{OutOfMemory};
 pub const Log2Align = math.Log2Int(usize);
 
-// The type erased pointer to the allocator implementation
-ptr: *anyopaque,
+/// The type erased pointer to the allocator implementation.
+/// If null, the allocator implementation has no associated state.
+ptr: ?*anyopaque,
```

Comment:
I'm not really happy with the wording of this doc comment. Suggestions are appreciated.

Reply:
I don't think there really needs to be a document comment for that, though if you really want I would suggest something like this,

```suggestion
/// A type erased pointer to the allocator's state
ptr: ?*anyopaque,
```

Reply:
I think why it's optional would be good to document, but am not sure what the best explanation would be.

Reply:
If you are not a Zig core team member, please refrain from posting your opinions on pull requests. Frankly, it is noise. If you have facts to share - for example, pointing out a bug, sharing a link to a technical specification, helping someone understand why the CI is failing, or other things based in objective reality - you are welcome to participate. Otherwise, silence please.

It goes deeper than my personal preference. New contributors to Zig, perhaps those less familiar with GitHub's particular quirks, find themselves facing opinions from random GitHub users and mistaking them for reviews by core team members.